### ISSUE TITLE
Assembly failed due to Graph alignment jobs failing

### ISSUE THREAD

**Issue by Lahcen86** [marbl/canu#656]
Hi there

Could anybody please shed some light on a recent failed assembly. Please see below for prerequisite information. 

I have had much success with Canu of late, producing multiple final assemblies with quality stats in the regions expected. However, I have now tried to tweak parameters to include more of our original data set by increasing over all coverage. I have allowed for 90X coverage initially up from the default 40X coverage. 

The assembly stage fails now, perhaps it is the parameters I have set... Im thinking maybe the minimum overlap length is too high maybe. My rational is to initially utilise more of our input reads, but selecting from a longer read length so increasing the default min read length to 2000. Although the 40X produced nice length stats etc, we miss many Orthologs following analysis with BUSCO2. Hence we want to try to incorporate more read data to mine for those missing portions of the genome likely sampled but never incorporated into the Canu assembly process. 

The error state given: ```**Graph alignment jobs failed, tried 2 times, giving up.**```

What confuses me is Canu seemingly provided the full length Assembly report with unitig overlap categories, N50 etc etc. So im not too sure why the finalised output files were not output too. No final contigs or anything like that, simply trimmed and corrected reads. See attached for report file. 
[Canu_report.txt](https://github.com/marbl/canu/files/1348735/Canu_report.txt)

Note this is the second attempt at this assembly, as I have tried to restart it from the first failure. Nothing was changed in those two runs, I simply resubmitted to our LSF cluster.

`Command used`: ```canu -p SSA1_SG1_Ug6SMRT-90X_Canu -d Canu_assembly_UgandSSA1-6SMRT_90Xmin -s ./canu-SSA1-SG1_Ug_6SMRT_runtimeSpecs_90x.spec```

`Specfile parameters set`:
```
#General run options:
shell=/usr/bin/bash
useGrid=true
gridEngine=lsf
genomeSize=0.6g
minReadLength=2000
gnuplot=/usr/bin/gnuplot

#grid options
maxThreads=48
maxMemory=128

correctedErrorRate=0.030
corOutCoverage=90
corMinCoverage=4
corPartitions=48

minOverlapLength=800 
unitigger=bogart


merylMemory=48
merylThreads=20

Canu snapshot v1.5 +180 changes (r8380 0ef11bf4114321684aa57cab4cb8d39c7c6aee96)
```
If any one has any suggestions I would very much appreciate it :) Thank you in advance ! 

Kind regards,
Lahcen

**Comment by skoren** [comment]
There should be more information in the ```unitigging/4-unitigger/alignGFA.1.out``` file. Most likely, it didn't request enough memory for the process. You can try increasing the memory using the ```gfaMemory``` parameter.

**Comment by Lahcen86** [comment]
Hi Skoren

Thank you very much for your tip. I found the LSF outfile reporting low mem indeed ! I promise I looked really hard for that file but in the directories etc I never located that. Perhaps in the future, offending report files can be sym linked to the main analysis directory. 

Was able to restart it with increased RAM needs. Assembly completed fully and output files as expected !! 

Thanks again
L
